src/factories/disputeBondTokenFactory.se

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+
+extern delegateExtern: [setup:[int256,int256]:_]
+extern disputeBondToken: [initialize:[address,address,uint256,int256]:bool]
+
+# workaround for https://github.com/ethereum/serpent/issues/119
+data lastDisputeBondToken
+
+def createDisputeBondToken(controller: address, market: address, bondHolder: address, bondAmount: int256, payoutDistributionHash: int256):
```

Comment:
bondAmount I think should be uint256. Not really an attack vector just for consistency.